### PR TITLE
Refactored code for directory changes

### PR DESCRIPTION
**Pull request by Geek-ubaid**
- All the directory issues are resolved
- No bug in local installation
- Installation instructions added in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,22 +52,37 @@
...
 * Pre-requisites:
-  - Python version > 3
+  - Python version >=3.4
   
 * Install the package using 
-```bash
-pip install pipgui==1.1
-```
+  ```bash
+  pip install pipgui==1.3
```
Comment (r00tdaemon) [inline README.md]: Why should the users specify a version number?
they should just get the latest version

Comment (Geek-ubaid) [reply README.md]: okay ill remove that.

Comment (Geek-ubaid) [reply README.md]: removed version number from the readme

**Comment by r00tdaemon** [comment]
Please squash the commits before merging

**Pull request merged**